### PR TITLE
LPS-125592 As an account administrator, I want my account's tax exemption status stored

### PR DESCRIPTION
**Pull request by alee8888**
@drewbrokke

https://issues.liferay.com/browse/LPS-125592

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-service/src/main/java/com/liferay/account/internal/upgrade/v1_3_0/UpgradeAccountEntry.java [hunk 1]
```diff
@@ -41,6 +41,12 @@ protected void doUpgrade() throws Exception {
 				AccountEntryTable.class,
 				new AlterTableAddColumn("emailAddress", "VARCHAR(254) null"));
 		}
+
+		if (!hasColumn("AccountEntry", "taxExemptionCode")) {
+			alter(
+				AccountEntryTable.class,
+				new AlterTableAddColumn("taxExemptionCode", "String null"));
```
Comment (drewbrokke) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/internal/upgrade/v1_3_0/UpgradeAccountEntry.java:48]: I think `String` should be all caps: `STRING`

Comment (pei-jung) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/internal/upgrade/v1_3_0/UpgradeAccountEntry.java:48]: The `com.liferay.account.internal.upgrade.v1_3_0.util.AccountEntryTable` also needs to be regenerated.

**Comment by alee8888** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:forward

**Comment by Sharryxue** [comment]
ci:test:stable

**Comment by Sharryxue** [comment]
ci:test:relevant

**Comment by Sharryxue** [comment]
Retry the relevant tests to see if the OOM issue is consistant.

**Pull request closed without merging**